doc/articles/getting-started/wizard/projects.md

```diff
@@ -0,0 +1,8 @@
+---
+uid: Uno.GettingStarted.UsingWizard.Projects
+---
+
+- **Server**  
+    Adds an ASP.NET Core Server projects to the solution, which hosts the WASM project, and can also be used to create an API and endpoints. It can also be used as the data server and you can also choose to implement the authentication server code in it.
```

Comment:
```suggestion
    Adds an ASP.NET Core Server project to the solution, which hosts the WASM project, and can also be used to create an API and endpoints. It can also be used as the data server and you can also choose to implement the authentication server code in it.
```